modules/dxp/apps/search-experiences/search-experiences-rest-impl/src/main/java/com/liferay/search/experiences/rest/internal/jaxrs/exception/DuplicateSXPBlueprintExternalReferenceCodeExceptionMapper.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ *
+ *
+ *
+ */
+
+package com.liferay.search.experiences.rest.internal.jaxrs.exception;
+
+import com.liferay.portal.vulcan.jaxrs.exception.mapper.BaseExceptionMapper;
+import com.liferay.portal.vulcan.jaxrs.exception.mapper.Problem;
+import com.liferay.search.experiences.exception.DuplicateSXPBlueprintExternalReferenceCodeException;
+
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author Gustavo Lima
+ */
+@Component(
+	enabled = false,
+	property = {
+		"osgi.jaxrs.application.select=(osgi.jaxrs.name=Liferay.Search.Experiences.REST)",
+		"osgi.jaxrs.extension=true",
+		"osgi.jaxrs.name=Liferay.Search.Experiences.REST.DuplicateSXPBlueprintExternalReferenceCodeExceptionMapper"
+	},
+	service = ExceptionMapper.class
+)
+public class DuplicateSXPBlueprintExternalReferenceCodeExceptionMapper
+	extends BaseExceptionMapper
+		<DuplicateSXPBlueprintExternalReferenceCodeException> {
+
+	@Override
+	protected Problem getProblem(
+		DuplicateSXPBlueprintExternalReferenceCodeException
+			duplicateSXPBlueprintExternalReferenceCodeException) {
+
+		return new Problem(
+			Response.Status.BAD_REQUEST, "Duplicate external reference code");
+	}
+
+	@Override
+	protected boolean isSanitize() {
```

Comment:
Reviewing I realized that it should not be overridden. Someone change the BaseExceptionMapper to false and did not remove the uses that are overriding it. Should we open a new ticket for that? 
@joshuacords

Reply:
I think we should actually leave it as is, there's no guarantee that method's logic might change - but we do know this problem isn't sanitized. Additionally, there's a bunch of other subclasses that still override it with `return false` and they're still there, so we are just following a pattern.